Reorganizing the current pages and ensuring all pages exist outside of reference. (#151)
aerleon/aerleon: docs/user/faq.md

# Frequently Asked Questions

> ## What does Aerleon mean?

Reviewer: ```suggestion
## What Does Aerleon Mean?
```
Title Case

> Aerleon is a fork of the Capirca project. This project was intended to be splled Caprica after the planet in Battlestar Galactica. There was a transposition error when first releasing the project to the public. To pay respect to where this fork came from we have chosen a different planet from Battlestar Galatica and made a similar error.

Reviewer: ```suggestion
Aerleon is a fork of the Capirca project. This project was intended to be spelled Caprica after the planet in Battlestar Galactica. There was a transposition error when first releasing the project to the public. To pay respect to where this fork came from we have chosen a different planet from Battlestar Galactica and made a similar error.
```

> ## How do you pronounce these projects names?

Reviewer: ```suggestion
## Pronunciation?
```
